mantidproject/mantid
# Update GPSANS instrument definition file

**Description of work.**
- [x] Replaced `CG2_Definition.xml`
- [x] Take into account the difference in sequence of tube ID's between the XML file and the IDF.
- [x] Updated [release notes](https://github.com/mantidproject/mantid/blob/0103862326ff4c7fb3692f96b337886cdc76b315/docs/source/release/v4.2.0/sans.rst#id5)

**To test:**  
Run the following code in two versions of MantidPlot. One containing the old `CG2_Definition.xml` file and the other containing the new version in this branch.
```
data_dir = '/SNS/EQSANS/shared/sans-backend/data/new/ornl/sans/hfir/gpsans/'
w = LoadHFIRSANS(data_dir + 'CG2_exp206_scan0016_0001.xml')
```
Then  show the instrument in the "Full 3D" view with the "Z-" projection. Using mantid/master, the beam center should be located "down and to the right" of the origin. Using mantid/this_branch, the beam center will be located "down and to the left" of the origin. This corrects the X to -X inversion incurred by the IDF currently stored in mantid/master

Fixes #26710

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
